addons.tf

```diff
@@ -202,3 +202,28 @@ module "cluster_secret_store" {
     module.addons
   ]
 }
+
+################################################################################
+# Reloader
+module "reloader" {
+  source = "./modules/addon"
+  create = var.create_addons && var.enable_reloader
```

Comment:
add a newline to make it a little more readable

Reply:
Okey